zoloto/coords.py

```diff
@@ -108,85 +177,123 @@ def from_cartesian(cls, cartesian: CartesianCoordinates) -> SphericalCoordinates
 class Orientation:
     """The orientation of an object in 3-D space."""
 
+    # The rotation so that (0, 0, 0) in _yaw_pitch_roll is a marker facing
+    # directly at the camera, not away
+    __MARKER_ORIENTATION_CORRECTION = Quaternion(matrix=np.array([
+        [-1, 0, 0],
+        [0, 1, 0],
+        [0, 0, -1],
+    ]))
+
     def __init__(self, e_x: float, e_y: float, e_z: float):
         """
         Construct a quaternion given the components of a rotation vector.
 
-        More information: https://w.wiki/Fci
+        Exposes the principal rotations as attributes.
+
+        More information: https://w.wiki/3FES
         """
         rotation_matrix, _ = Rodrigues((e_x, e_y, e_z))
-        self._quaternion = Quaternion(matrix=rotation_matrix)
+        initial_rotation = Quaternion(matrix=rotation_matrix)
+        # Remap axis to get conventional orientation for yaw, pitch & roll
+        # and rotate so that 0 yaw is facing the camera
+        quaternion = Quaternion(
+            initial_rotation.w,
+            -initial_rotation.z,
+            -initial_rotation.x,
+            initial_rotation.y,
+        ) * self.__MARKER_ORIENTATION_CORRECTION
+
+        if os.environ.get('ZOLOTO_LEGACY_AXIS'):
+            self._quaternion = initial_rotation
+        else:
+            self._quaternion = quaternion
+
+        self._yaw_pitch_roll = quaternion.yaw_pitch_roll
```

Comment:
Question: Is it intentional not to use `self._quaternion` here? This won't change in legacy axis mode

Reply:
Good point, and whilst this would be a breaking change, it's strictly an improvement rather than an convention change.